Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 2.0.1
-appVersion: 2.0.5
+version: 2.1.0
```

Comment:
This is something that could be configured in the `values.yml` as well. Allowing to upgrade the chart without upgrading traefik ?

Reply:
This comment is especially relevant provided the version below is a release candidate :)